packages/xsnap/package.json

```diff
@@ -14,7 +14,9 @@
     "repl": "node src/xsrepl.js",
     "build:bundle": "rollup --config rollup.config.js",
     "build:bin": "node src/build.js",
-    "build": "yarn build:bin && yarn build:bundle",
+    "build:env": "node src/build.js --show-env > dist/build-env",
+    "build:from-env": "{ cat dist/build-env; echo node src/build.js; } | xargs env",
```

Comment:
Probably for another time, but it would be great if we could at least distribute the source so that we don't have a reliance on `git` here. Even better would be to build the binaries at install time, which is a little more node like.